### ISSUE TITLE
DocsAlive

### ISSUE THREAD

**Issue by julo283** [MozillaFestival/open-leaders-7#85]
Project Lead: julo283

Mentor: fayetandog

Welcome to OL7, Cohort A! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***

Before Week 1 (Jan 29): Your first mentorship call
- [x] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Feb 5): First Cohort Call (Open by Design)
- [x] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [x] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [x] Check the [Syllabus](https://mzl.la/ol7-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 12): Mentorship call
- [x] Look up two other projects and comment on their issues with feedback on their vision statement.
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [ ] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Feb 19): Cohort Call (Build for Understanding)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.
- [x] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.
![image](https://user-images.githubusercontent.com/47010177/52947962-1a2cad00-3370-11e9-9757-e8431db2c788.png)

Week 5 and more
- [ ] Create a repository for your project. New to GitHub? You can [fork our templates](https://mozilla.github.io/global-sprint/project-lead-guide/templates/).
- [ ] Add [Contributing Guidlines](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/)
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)
- [ ] [Submit your project to Mozilla Pulse](https://www.mozillapulse.org/add), add the tag `#mozsprint`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Leaders. Please refer to the [OL7 Syllabus](https://mzl.la/ol7-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by julo283** [comment]
Vision/Mission statement draft: 
I’m working with members of the citizen science community to create resources sharing knowledge, best practice and other kinds of insights around citizen science and education so that we can form a community of practice that is open, safe, diverse & synergistic.

**Comment by julo283** [comment]
Open canvas for this project [https://docs.google.com/presentation/d/1Gya_Tg1Zg_y5ivvdDGwp8zpaN7k4qHLtHw4AVO5nY9E/edit#slide=id.p](url)

**Comment by jmtaylor86** [comment]
> Vision/Mission statement draft:
> I’m working with members of the citizen science community to create resources sharing knowledge, best practice and other kinds of insights around citizen science and education so that we can form a community of practice that is open, safe, diverse & synergistic.

Your vision statement is extremely clear and the project sounds interesting. Look forward to seeing some of the resources that get shared.

**Comment by annefou** [comment]
I agree: your visions statement is very clear. I cannot access your open canvas (I get an error: "No results matched your search."). I would be interesting in looking at it. Thanks.

**Comment by lsrkthelibrarian** [comment]
Last summer, while taking a course in GIS Mapping, I was fortunate to have an instructor that included citizen science techniques in the course: [Globe Observer](https://observer.globe.gov) It was a very valuable, and fun, part of the course.  I'd like to see more citizen science in all forms of education!  I'm excited to hear about your project.

**Comment by t2d** [comment]
Your canvas misses "Unique Value Proposition". I think it would be great to fill this to make clear what your project has that other approaches are missing.
There are lots of channels and profiles. Is this all existing already? If not, maybe it helps to focus on how to engage a specific group?

**Comment by julo283** [comment]
My project [roadmap](https://docs.google.com/presentation/d/15lPo8a1lKfPg-pKkNRVGVhsam4eAbliHgIBy8-hRPYI/edit?usp=sharing)

**Comment by julo283** [comment]
> Your canvas misses "Unique Value Proposition". I think it would be great to fill this to make clear what your project has that other approaches are missing.
> There are lots of channels and profiles. Is this all existing already? If not, maybe it helps to focus on how to engage a specific group?

Thank you very much for your input @t2d ! I needed more time to think about the UVP, but have added it now. 
The whole project is about reaching out to all those groups and providing a way to participate for people not attending the few physical meetings we have. I see this as a prototype and hope to be able to demonstrate that something like this is needed and how it could work. After that ideally, something similar would be implemented in a not-for-profit platform that has been funded but is in very early days.
